html/selionGrid.html

```diff
@@ -15,9 +15,9 @@ <h2 id="introduction">Introduction</h2>
         PhantomJS binary in each node without impacting existing test case execution.</li>
     <li>Force Restart Node – Forcibly restart the node without logging in to the
         physical node.</li>
-    <li>Self-Healing – After execution of 25 test cases, the grid node will be recycled
+    <li>Self-Healing – After execution of 50 test cases, the grid node will be recycled
```

Comment:
`50` is the default value and it's completely configurable. We need to be a bit clearer with respect to this. Something like;

`Self-Healing - Recycles the node after a pre-defined number of unique sessions are finished. Defaults to 50. Configurable via <code>$SELION_HOME/config/SeLionConfig.json</code>`